### PR TITLE
Change/CATX A1 SLDT 255 Add Help Links

### PR DESCRIPTION
**Pull request by JaneMcBrain**
https://jira.catena-x.net/browse/A1SLDT-255

### CODE REVIEW

File: portal/code/tractus-x-portal/src/components/navigation/Pagination.tsx [hunk 1]
```diff
@@ -5,16 +5,18 @@ export default function Pagination(props){
   const ChevronLeft: IIconProps = { iconName: 'ChevronLeft' };
   const ChevronRight: IIconProps = { iconName: 'ChevronRight' };
   const [isDisabledBefore, setIsDisabledBefore] = useState<boolean | any>(false);
+  const [isDisabledAfter, setIsDisabledAfter] = useState<boolean | any>(false);
```
Comment (drcgjung) [inline portal/code/tractus-x-portal/src/components/navigation/Pagination.tsx:8]: Could it be that there is another pagination change incorporated or rather missed in our branch?

File: portal/code/tractus-x-portal/src/components/semantics/SemanticHub.tsx [hunk 2]
```diff
@@ -57,7 +57,7 @@ export default class SemanticHub extends React.Component<any, any>{
       error: null,
       currentPage: defaultPage,
       pageSize: defaultPageSize,
-      hasNoNextPage: false
+      totalPages: 1
```
Comment (drcgjung) [inline portal/code/tractus-x-portal/src/components/semantics/SemanticHub.tsx:60]: same as above, this relates to pagination, not the stated ticket, right?

**Review by drcgjung — COMMENTED** [review]
Shouldn't we merge main to the feature branch before?
Problem is that there is some change (fetch api for twin registry) pending not yet in main.

File: portal/code/tractus-x-portal/src/components/navigation/Pagination.tsx (hunk 1 above)
Comment (JaneMcBrain) [reply portal/code/tractus-x-portal/src/components/navigation/Pagination.tsx:8]: yes, there is something wrong here... will fix it.

Comment (JaneMcBrain) [reply portal/code/tractus-x-portal/src/components/navigation/Pagination.tsx:8]: So it seems that the branch: feature/CATX-A1-SLDT doesn't have the pagination feature (which is already in th main branch).

**Review by drcgjung — APPROVED** [review]
merged with main before. looks better

**Pull request merged**